tests/e2e/fixtures/storage_efs_dependencies.py

```diff
@@ -211,11 +224,14 @@ def on_delete():
                 MountTargetId=mount_target_id,
             )
 
+        wait_on_mount_target_status("deleted", efs_client, fs_id)
+
         # Delete the Filesystem
+
         efs_client.delete_file_system(
             FileSystemId=fs_id,
         )
-        wait_on_efs_status("deleted", efs_client, fs_id)
+        wait_on_efs_status("deleting", efs_client, fs_id)
```

Comment:
why not "deleted" anymore?

Reply:
because deleted gets missed since it is a short lived status and soon goes into error state. However I can fix this similar to fsx and wait on the error and make this cleaner. Let me do that.

Reply:
changed to wait on completion of deletion instead same as fsx.